lib/shared/drive-constraints.js

```diff
@@ -208,3 +208,85 @@ exports.isDriveValid = (drive, image) => {
 exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE);
 };
+
+const feedbackMessages = {
```

Comment:
If unsafe mode is enabled, IMHO flashing a system drive should be downgraded from an error to a warning, and not simply treated as "not an error". But I have no preferences on the logic / mechanisms used to achieve that.